docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -121,6 +121,8 @@ NOTE: The timepicker on the right side of the page allows you to set a time inte
 
 Administrators can add Windows, macOS, and Linux applications that should be trusted. By adding these "trusted applications," you can use {elastic-sec} without compatibility or performance issues with other installed applications on your system. Trusted applications are applied only to hosts running {endpoint-sec}.
 
+Trusted applications are designed to help mitigate performance issues and incompatibilities with other endpoint software. However, they create blindspots for the the security solution. One avenue attackers use to exploit these blindspots is with "dll-side loading", where they leverage processes signed with trusted vendors (e.g. antivirus software) to execute their malicious dynamic link library. Such activity appears to originate from the trusted vendor's process.  To minimize the gaps in visibility to the security solution, be as specific as possible while creating entries. For example, combine `Signature` information with known `File Paths`.
+
```

Comment:
Two spaces before To minimize

Reply:
https://github.com/elastic/security-docs/pull/810/files#r670538923

Reply:
err https://github.com/elastic/security-docs/pull/810/files#r670540113